src/containers/Preview.js

```diff
@@ -27,7 +29,27 @@ function mapStateToProps(state) {
   };
 }
 
+function generateConsoleLogDispatcher(dispatch, timeout) {
+  let queue = [];
+
+  function flushQueue() {
+    dispatch(consoleLogBatchProduced(queue));
+    queue = [];
+  }
+
+  const timer = throttle(flushQueue, timeout, {
```

Comment:
I don’t think `timer` really describes it, maybe something like `throttledFlushQueue`, `flushQueueWithThrottling`, `flushQueueSoon`, etc.?